protocol/avdl/keybase1/kbfs_common.avdl

```diff
@@ -51,13 +51,30 @@ protocol kbfsCommon {
     UID writerUid;
     Time localTime;
     FolderType folderType;
-}
+  }
 
   record FSEditListRequest {
     Folder folder;
     int requestID;
   }
 
+  record FSFolderWriterEdit {
+    string filename;
+    FSNotificationType notificationType;
+    Time serverTime;
+  }
+
+  record FSFolderWriterEditHistory {
+    string writerName;
+    array<FSFolderWriterEdit> edits;
+  }
+
+  record FSFolderEditHistory {
+    Folder folder;
+    Time serverTime;
+    array<FSFolderWriterEditHistory> history;
```

Comment:
Is there a reason why this isn't a map? It probably won't make much a difference and I'm just curious.

Reply:
> Is there a reason why this isn't a map? It probably won't make much a difference and I'm just curious.

Sorry, I forgot to answer this.  It's not a map because it needs to be ordered, with the most recent edits first.